test_data/import_publications.py

```diff
@@ -20,13 +18,18 @@
                                  WithDecryption=False)['Parameter']['Value']
 CLIENT_ID = ssm.get_parameter(Name='/CognitoUserPoolAppClientId',
                               WithDecryption=False)['Parameter']['Value']
+user_tablename = ssm.get_parameter(Name='/test/USER_TABLE',
+                                   WithDecryption=False)['Parameter']['Value']
 publication_template_file_name = './publications/new_test_registration.json'
 test_publications_file_name = './publications/test_publications.json'
-user_tablename = 'nva-users-and-roles-nva-identity-service-nva-identity-service'
 person_query = 'https://api.{}.nva.aws.unit.no/person/?name={} {}'
 user_endpoint = 'https://api.{}.nva.aws.unit.no/users-roles/users/{}'
 upload_endpoint = 'https://api.{}.nva.aws.unit.no/upload/{}'
-publication_endpoint = 'https://api.{}.nva.aws.unit.no/publication'.format(STAGE)
+publication_endpoint = 'https://api.{}.nva.aws.unit.no/publication'.format(
+    STAGE)
+publish_endpoint = 'https://api.{}.nva.aws.unit.no/publication/publish/{}'
+request_doi_endpoint = 'https://api.{}.nva.aws.unit.no/publication/doirequest'.format(
+    STAGE)

```

Comment:
fyi så støtter python stort sett samme greia som JS for string interpolation: https://docs.python.org/3/tutorial/inputoutput.html#fancier-output-formatting
så man kan evt skrive det slik også

```suggestion
request_doi_endpoint = f'https://api.{STAGE}.nva.aws.unit.no/publication/doirequest'
```